SwiftLeeds/Views/Components/FancyHeaderView.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  HeaderView.swift
+//  SwiftLeeds
+//
+//  Created by Kannan Prasad on 05/07/2022.
+//
+
+import SwiftUI
+
+struct FancyHeaderView: View {
+    let title: String
+    let foregroundImageURL: URL
+    let backgroundImageURL: URL
+    
+    private let foregroundImageWidth = 160.0
```

Comment:
I wonder if this would be better as a percentage of the screen rather than a fixed width?

Reply:
So its a good concept _but_ I've always found doing it that way can end up having so many "if"s that its not necessarily worth it. The design is very much iPhone optimised, but ideally most things should still look fine on the iPad 👍🏻 

For iPhones, the end image would end up just being this number but with odd bits on the end, like say 168 or 154. Then on bigger devices if you have a 1:1 aspect ratio, the square would end up being absolutely gigantic. There's room to improve for sure, but I don't think this is the way - we'd be better going for explicit iPad layout optimisations based on environment.

Reply:
I've kept this as hard coded for now, as this was mentioned specifically on the designs. Let me know if the needs modifications.